config/application.rb

```diff
@@ -52,6 +52,12 @@ class Application < Rails::Application
       }
     end
 
+    config.good_job.cron['autoload_fs_boxes'] = {
+      cron: "00 6 * * *",  # run every day at 6:00 am
+      class: "Fs::BoxifyAllApiConnectionsJob",
+      description: "Regular job to autoload FS boxes"
```

Comment:
Nie som uplne presvedceny, ze toto chceme. Dava mi zmysel to zavolat hned ked sa vytvori fs connection ale preco kazdy den?

Reply:
Je to na zaklade feedbacku od klienta. Tie schranky priebezne pribudaju nasim klientom, ak to je napr. vacsia uctovna kancelaria a pribudne im novy klient. Uz sme sa s tym stretli, prisla poziadavka, nech dotiahneme nove schranky do GO.

Reply:
A ked nejaka ubudne?

Reply:
To je trochu komplikovanejsi scenar, zatial sme neriesili. Mazat len tak automaticky schranky bez upozornenia urcite nechceme. Postupne sa dostaneme aj k tomuto pripadu, zistime podla mna, co presne sa ma udiat (predpokladam, ze len znefunkcnit nahravania/stahovanie sprav, ale schranku spolu s historiou nemazat automaticky).